pkgs/development/tools/coder/default.nix

```
@@ -1,8 +1,18 @@
-{ buildGoModule
+{ lib
 , fetchFromGitHub
 , installShellFiles
-, lib
+, makeWrapper
+, buildGoModule
+, fetchYarnDeps
+, fixup_yarn_lock
+, pkg-config
+, nodejs
+, yarn
+, nodePackages
+, python3
+, terraform
```

probably good to stort these:
```suggestion
, fetchFromGitHub
, installShellFiles
, makeWrapper
, buildGoModule
, fetchYarnDeps
, fixup_yarn_lock
, pkg-config
, nodejs
, yarn
, nodePackages
, python3
, terraform
```

---

done :)